libs/testing/e2e/src/lib/support/wire-mocks.ts

```
@@ -0,0 +1,170 @@
+import {
+  EqualPredicate,
+  FlexiPredicate,
+  HttpMethod,
+  Imposter,
+  Mountebank,
+  Operator,
+  Proxy,
```

Biome

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)